cypress/e2e/specialFeatures/atiAnalytics/index.cy.js

```diff
@@ -283,6 +285,65 @@ const canonicalTestSuites = [
       assertBillboardComponentClick,
     ],
   },
+  {
+    path: '/persian/topics/crezq2dg9zwt',
+    runforEnv: ['local', 'test'],
+    service: 'persian',
+    pageIdentifier: 'persian.topics.crezq2dg9zwt.page',
+    applicationType: 'responsive',
+    contentType: 'index-category',
```

Comment:
I've just found this... I think this means there's a mismatch between page & event tracking for topic pages (which has been there for a longggg time 😬 ). We should check with product what is the correct value

https://github.com/bbc/simorgh/blob/c4a637795cd265fb4e911fd40dab7025eb04659c/src/app/contexts/EventTrackingContext/index.tsx#L66

Reply:
This has been fixed in the PR https://github.com/bbc/simorgh/pull/12493. We needed to keep 'topic-page' for component level tracking and 'index-category' for page level tracking.